### PR TITLE
automate extension tests

### PR DESCRIPTION
**Pull request by aykut-bozkurt**

### CODE REVIEW

File: extensionQueries/hll.sql [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+FROM facts
+GROUP BY 1;
+
+SELECT EXTRACT(MONTH FROM date), hll_cardinality(hll_union_agg(users))
+FROM daily_uniques
+GROUP BY 1;
+
+SELECT date, #hll_union_agg(users) OVER seven_days
+FROM daily_uniques
+WINDOW seven_days AS (ORDER BY date ASC ROWS 6 PRECEDING);
```
Comment (gledis69) [inline extensionQueries/hll.sql:52]: NIT: missing `\n` at the end of the file

File: extensionQueries/topn.sql [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+FROM popular_products
+WHERE review_date >= '2000-01-01' AND review_date < '2000-02-01'
+ORDER BY 2 DESC;
+
+SELECT date_trunc('month', review_date) AS review_month,
+       (topn(topn_union_agg(agg_data), 1)).*
+FROM popular_products
+WHERE review_date >= '2000-01-01' AND review_date < '2001-01-01'
+GROUP BY review_month
+ORDER BY review_month;
```
Comment (gledis69) [inline extensionQueries/topn.sql:48]: NIT: missing `\n`

**Comment by gledis69** [comment]
Can you push this PR on a branch which is called `extension/...` so I can have a look at the test triggering and the operations being performed and the result generation (if there is any).

**Comment by gledis69** [comment]
@onurctirtir have you ever done manual extension testing. If yes, can you have a look at the queries that Aykut is running for the extension tests. If not, we should have smn else have a look.

**Comment by onurctirtir** [comment]
> @onurctirtir have you ever done manual extension testing? If yes, can you have a look at the queries that Aykut is running for the extension tests. If not, we should have smn else have a look.

I will have a look into the PR within a few days

**Comment by aykut-bozkurt** [comment]
I am closing that PR and moving it on another branch prefixed with '**extension**' so that any push triggers a job to investigate. Missing \n comments are not applicable anymore because we will be running extensions' own schedules. (not the ones created by me)

**Pull request closed without merging**